### ISSUE TITLE
All popups to have links (with preserved settings)

### ISSUE THREAD

**Issue by ghost** [Chocobozzz/PeerTube#2930]
I'd like to share the 'Settings' page of an instance, but when I click the 'Settings' button the link doesn't change. I'd also like the link to change when I pick different settings, for example, if I click on the 'Settings' button and then select 'Display' for 'Default policy on videos containing sensitive content' then the link should look something like this https://abc.com/settings&policy=display, if I also select 'English' from the 'Only display videos in the following languages/subtitles' then https://abc.com/settings&policy=display&lang=eng

The 'Contact administrator' button also doesn't have a link when clicked. Same for 'Shortcuts' and perhaps others as well.

**Comment by Chocobozzz** [comment]
Implemented as a plugin hook in https://github.com/Chocobozzz/PeerTube/commit/071f3e519cbd3184b59ee728fe96c5c29b7792b9
